Flipper/Packages/Peripheral/Sources/RPC/Model/Main/OutgoingMessage.swift

```diff
@@ -0,0 +1,39 @@
+public enum OutgoingMessage {
+    case screenFrame(ScreenFrame)
+    case reboot(RebootMode)
+
+    public enum RebootMode {
+        case os
+        case dfu
+        case update
+    }
+}
+
+extension PBSystem_RebootRequest.RebootMode {
+    init(_ source: OutgoingMessage.RebootMode) {
+        switch source {
+        case .os: self = .os
+        case .dfu: self = .dfu
+        case .update: self = .update
+        }
```

Comment:
Because of Swift 5.9, now you can make it simpler

```suggestion
        self = switch source {
        case .os: .os
        case .dfu: .dfu
        case .update: .update
        }
```